qa/rpc-tests/importmulti.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2016 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+class ImportMultiTest (BitcoinTestFramework):
+    def setup_chain(self):
+        print("Initializing test directory "+self.options.tmpdir)
+        initialize_chain_clean(self.options.tmpdir, 2)
```

Comment:
Could you use `__init__` for that, instead of overwriting `setup_chain`?

I.e. something like
```py
    def __init__(self):
        super().__init__()
        self.num_nodes = 2
        self.setup_clean_chain = True